### DISCUSSION TITLE
Update RetroArch to 1.19.1 and update cores to match

### DISCUSSION THREAD

**Discussion by cmclark00** [OnionUI/Onion#1614] [Ideas]
There have been a ton of improvements to RetroArch and quite a few of the cores since RA 1.15 released. Supafaust has retroachievements now, gpsp has rumble, as well as wireless link support for pokemon and other games. I know it may be low priority, but updating these things would add a decent chunk of features all at once. I tried to compile  them myself, and while both supafaust and gpsp worked on the old RA, retroachievements did not, and on gpsp the rumble was on continuously, so that was a no go lol.

**Comment by Aemiii91** [comment]
It's already planned for V4.4

**Reply by cmclark00** [reply]
Hot dog! That's awesome! Thank you for the reply and reassurance, I'll be eagerly awaiting the 4.4 beta lol.

**Reply by horendus** [reply]
Cant wait to evolve my Kadabra!
